site/content/en/docs/user/kwokctl-snapshot/yaml-recording.demo

```diff
@@ -0,0 +1,31 @@
+# Let's getting started with kwokctl!
```

Comment:
Using "Let's" with "getting" isn't grammatically correct here. You can either use any of the below:
```
# Let's get started with kwokctl!
```
This expresses an excitement to educate a user about kwokctl.
You can also use the below:
```
# Getting started with kwokctl!
```
The second one expresses an educator introducing users to a beginner guide to kwokctl.

Since this is not a beginner guide to kwokctl, I'll suggest the below:
```
# Let's get started with kwokctl!
```